pyproject.toml

```diff
@@ -13,12 +13,14 @@ numpy = "^1"
 pandas = "^1"
 pydateinfer = "0.3.0"
 python-dateutil = ">=2.1"
-scipy = ">=1.5.4"
+scipy = ">=1.7.3"
```

Comment:
Curious about this: any specific reason for the upgrade?

Reply:
I was having issues getting tests to pass with earlier version, bumping v seemed to fix it